### PR TITLE
Remove details-polyfill and details styling

### PR DESCRIPTION
**Pull request by RoseSAK**
## What

[Link to story](https://dsdmoj.atlassian.net/browse/AP-XXX)

We do not need details-polyfill as we are no longer supporting older browsers, we do not need the details white text class as this is no longer used

## Checklist

Before you ask people to review this PR:

- Tests and rubocop should be passing: `bundle exec rake`
- Github should not be reporting conflicts; you should have recently run `git rebase main`.
- There should be no unnecessary whitespace changes. These make diffs harder to read and conflicts more likely.
- The PR description should say what you changed and why, with a link to the JIRA story.
- You should have looked at the diff against main and ensured that nothing unexpected is included in your changes.
- You should have checked that the commit messages say why the change was made.

### CODE REVIEW

**Review by agoldstone93 — APPROVED** [review]

File: app/forms/providers/confirm_dwp_non_passported_form.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+module Providers
```
Comment (naseberry) [inline app/forms/providers/confirm_dwp_non_passported_form.rb:1]: Change doesn't seem related to the commit, was this meant to be committed?

**Review by agoldstone93 — APPROVED** [review]
